lib/timeutils/tests/test_scan-timestamp.c

```diff
@@ -293,6 +295,38 @@ Test(parse_timestamp, rfc5424_timestamps)
   _expect_rfc5424_timestamp_eq("2017-06-14T23:57:27Z", "2017-06-14T23:57:27.000+00:00");
 }
 
+Test(parse_timestamp, rfc3164_performance)
+{
+  const gchar *ts = "Dec 14 05:27:22";
+  const guchar *data = (const guchar *) ts;
+  gint length = strlen(ts);
+  WallClockTime wct = WALL_CLOCK_TIME_INIT;
+  gint it = 1000000;
+
+  start_stopwatch();
+  for (gint i = 0; i < it; i++)
+    {
+      scan_rfc3164_timestamp(&data, &length, &wct);
+    }
+  stop_stopwatch_and_display_result(it, "RFC3164 timestamp parsing speed");
```

Comment:
What is the purpose of such "test" ? if we could call it a test without any assertion, time limitation.

Reply:
This was addressed by excluding these performance tests during unit test invocation using a criterion environment variable. Still, if I (or anyone) runs the test manually, they get executed.